frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -358,19 +388,35 @@ function ReaderWikipedia:onLookupWikipedia(word, box, get_fullpage, forced_lang)
         })
     end
 
-    -- Fix lookup message to include lang
+    -- Fix lookup message to include lang and set appropriate error texts
+    local no_result_text, req_failure_text
     if get_fullpage then
         self.lookup_msg = T(_("Getting Wikipedia %2 page:\n%1"), "%1", lang:upper())
```

Comment:
Downloading Wikipedia page

Reply:
Is "Fetching Wikipedia page" ok ? (For me, "Downloading" has the implicit meaning I'll get a file down somewhere :)
And we should be consistent with Wikipedia "article" or "page" (I'm used to "page", dunno if "article" is preferable).

Reply:
I would say article or entry (as in a reference work), not page. Wikipedia calls it an article, so let's go with that.

https://en.wikipedia.org/wiki/Wikipedia:Your_first_article

Reply:
Fetching sounds a bit like jargon I'd say. Pinging @KenMaltby

Reply:
One of the aspects of the issue of selecting terms, for an international project like this, is that some of the subtleties are likely to be lost in translation.  So it may be best to try and find the simplest, most unambiguous term.  "Retrieving" should work well as its English meaning is a more general function that would include "Downloading" (and a dog fetching) something.  Hopefully, the auto translations well reflect that.